ads/_config.js

```diff
@@ -506,6 +506,10 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  kuadio: {
+    renderStartImplemented: true,
```

Comment:
are you sure you are firing this method in your script? I couldn't get the listener to activate

Reply:
Guidelines suggests that renderStart API implementation is optional. 

Could We implement renderStart API in a near future?

Reply:
It's true, it is up to you whether or not to implement. If you are not going to implement with this PR just remove `renderStartImplemented` and you can add it with a later PR when ready.